ui/src/components/group/GroupRow.js

```diff
@@ -189,9 +204,15 @@ class GroupRow extends React.Component {
             ) : (
                 <span>{' ' + this.state.name}</span>
             );
+        let newGroupAnimation =
+            this.props.domain + '-' + this.state.name === this.props.newGroup;
```

Comment:
wont this return <domain>-true/false?

Reply:
yup this is the flag whether to show the animation for a specific row